spec/controllers/suscribir/suscripciones_controller_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+module Suscribir
+  describe SuscripcionesController do
+    routes { Suscribir::Engine.routes }
+
+    describe "desuscribir" do
+      let(:suscripcion) { create(:suscripcion) }
+
+      it "debe borrar la suscripcion si el token es correcto" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.